packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -388,9 +389,31 @@ class DebuggerController extends DisposableController
     // we fetch all variable objects twice (once in _getFullStack and once in
     // in_createStackFrameWithLocation).
     if (await serviceManager.connectedApp!.isDartWebApp) {
+      final topFrame = pauseEvent?.topFrame;
+      if (topFrame == null) {
```

Comment:
maybe we should add an analytics event here too so that we can catch this sort of thing in the future and measure the occurrence of this issue over time

Reply:
Done! Added a `pausedWithNoFrames` event